_posts/2019-01-03-elixir-kaffe-codealong.md

```diff
@@ -0,0 +1,236 @@
+---
+author: Meryl Dakin
+author_link: https://github.com/meryldakin
+categories: til
```

Comment:
@meryldakin / @SophieDeBenedetto thoughts on a category for this type of lesson? It's not a TIL. We aren't using categories yet but we've discussed using them. I think @burden may have started on something, if not I certainly can if we'd find it useful 😁

Reply:
ah yea good call. Yea definitely not a TIL 🤔 ...I would love to standardize some categories but off the top of my head we could do `code along`, `kaffe`, `kafka`?

Reply:
how about `libraries`? to hit the "using external libraries" category

Reply:
Otherwise this all looks done to me! Let's pick some categories and get this out 🚀